app/__tests__/fixtures/cluster-updates/cluster-updates-devnet.json

```diff
@@ -0,0 +1,47 @@
+{
+  "epochInfo": {
+    "absoluteSlot": {
+      "__type": "bigint",
+      "__value": "429308025"
+    },
+    "blockHeight": {
+      "__type": "bigint",
+      "__value": "417224560"
+    },
+    "epoch": {
+      "__type": "bigint",
+      "__value": "993"
+    },
+    "slotIndex": {
+      "__type": "bigint",
+      "__value": "332025"
+    },
+    "slotsInEpoch": {
+      "__type": "bigint",
+      "__value": "432000"
+    }
+  },
+  "epochSchedule": {
+    "firstNormalEpoch": {
+      "__type": "bigint",
+      "__value": "0"
+    },
+    "firstNormalSlot": {
+      "__type": "bigint",
+      "__value": "0"
+    },
+    "leaderScheduleSlotOffset": {
+      "__type": "bigint",
+      "__value": "432000"
+    },
+    "slotsPerEpoch": {
+      "__type": "bigint",
+      "__value": "432000"
+    },
+    "warmup": false
+  },
+  "firstAvailableBlock": {
+    "__type": "bigint",
+    "__value": "116113408"
+  }
+}
```

Comment:
Could you, please, ensure that all fixtures are created with proper symbol in the end. Looks like fixtures are not the target for prettier atm